xmtp_mls/src/groups/mod.rs

```diff
@@ -326,6 +328,9 @@ impl MlsGroup {
         let welcome = deserialize_welcome(&welcome_bytes)?;
 
         let join_config = build_group_join_config();
+        if join_config.number_of_resumption_psks > 0 {
```

Comment:
Just noting I had to make this public so I could use it https://github.com/xmtp/openmls/pull/30

Reply:
This isn't doing what you think it's doing I'm afraid. This only prevents you from joining a group that allow resumption PSKs. It's a property you probably want to set when creating groups to avoid generating unused resumption PSKs.

But to achieve what you want, you need to look into the group secrets. That's actually not possible yet. But it is with openmls/openmls#1585. I just added a function there to [get the psks](https://github.com/openmls/openmls/pull/1586/commits/703fb763a2eef40d401c9ecfcbc9bab188663b6a). On the result from that function you can get the `processed_welcome.psks().psk()`, which returns a [`PSK`](https://openmls.tech/openmls/doc/openmls/schedule/psk/enum.Psk.html) enum. There you can error out when it contains a resumption psk (or any, depending on what you want to achieve).

Reply:
I pulled in your PR to my fork and I seem to be able to get the psks now. 🙏